packages/babel-preset-env/src/normalize-options.js

```diff
@@ -297,5 +313,10 @@ export default function normalizeOptions(opts: Options) {
     spec: validateBoolOption(TopLevelOptions.spec, opts.spec, false),
     targets: normalizeTargets(opts.targets),
     useBuiltIns: useBuiltIns,
+    browserslistEnv: validateStringOption(
+      TopLevelOptions.browserslistEnv,
+      opts.browserslistEnv,
+      "production",
```

Comment:
Let's unset the default values here, otherwise it will be a breaking change.

Here is how `browserslist` picks its env https://github.com/browserslist/browserslist/blob/adcb789638f8d50fbdd6fe53db164e9cd4bb4444/node.js#L66

Basically it is `env > NODE_ENV > BROWSERSLIST_ENV > "production"`. Now if users are using the following config
```
[development]
last 1 chrome

[production]
last 1 version
```

Running `NODE_ENV=development babel --presets env` will have `browserslist` picked `development` as the `env`, however after the defaults are assigned here, `browserslist` will now pick `production` over `process.env.NODE_ENV`.

Reply:
Fair point